### PR TITLE
Fix Saving Messages

### PR DESCRIPTION
**Pull request by tikhsuP**
This PR ensures the following:
1. "Saving Snapshot..." and "Save Successful" messages can only be seen on the tab which is being/has been saved.
2. The toolbar icons ('S' and 'check') can only be seen on the tab which is being/has been saved.

Issue #530

### CODE REVIEW

**Comment by anishsarangi** [comment]
Looks fine the new changes :-)

File: scripts/background.js [hunk 1]
```diff
@@ -509,21 +512,25 @@ function updateWaybackCountBadge(tabId, url) {
  * Name string is based on PNG image filename in images/toolbar/
  * @param name {string} = one of 'archive', 'check', 'R', or 'S'
  */
-function setToolbarIcon(name) {
+function setToolbarIcon(tabId, name) {
```
[the remark this answers is not in the document]
Comment (tikhsuP) [reply scripts/background.js:515]: Sorry for that. 
I will make the required changes. Thanks !! 🙂

File: scripts/background.js [hunk 2]
```diff
@@ -523,7 +526,11 @@ function setToolbarIcon(name) {
 
 function setToolbarState(tabId, name) {
   toolbarIconState[tabId] = name
```
Comment (tikhsuP) [inline scripts/background.js:528]: @cgorringe @anishsarangi I would like to know the best place for this line. Should it be placed inside the **if ( )** statement, Line No. 530?

 `if (tabs[0].id === tabId) {
      setToolbarIcon(name)
    }`

I couldn't find any differences while testing the extension.

**Comment by cgorringe** [comment]
Sorry this took me awhile to get to. I made additional fixes after reviewing the code. Works great now, even with multiple Saves happening concurrently. You can go ahead and squash and merge to master @tikhsuP

**Pull request merged**